### ISSUE TITLE
Show message next to pagination that search needs to be refined or go to biblio front-office

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#727]
# Story

Whenever I reach the end of the 20 pages that I got back from my search result, I am encouraged to refine my search query or go to the front-office.

To be implemented with #708 

## Screenshots
![Image](https://user-images.githubusercontent.com/1572885/197235522-4a6838cc-77cd-4587-8151-1a62913f4d56.png)

## Implementation suggestion
```
<div class="alert alert-info">
    <i class="if if-info-circle-filled"></I>
    <div>
        <h4>Biblio search results have been optimised</h4>
        <p class="mb-2">
          To optimise Biblio's speed, we only show you the most relevant records.
          <br>Missing some results?
        </p>
        <ul class="mb-2">
          <li><strong>Refine your search query</strong> and use the <strong>filters</strong> to get a more precise set of record.</li>
          <li>Go to <a href="http://biblio.ugent.be" target="_blank">http://biblio.ugent.be</a> and <strong>use the extended search</strong> to get a complete picture.</li>
        </ul>
    </div>
</div>
```

**Comment by mietcls** [comment]
We can do this much easier. Add an icon to the end of the pagination...

**Comment by mietcls** [comment]
To keep information

# Story

Whenever I reach the end of the 20 pages that I got back from my search result, I am encouraged to refine my search query or go to the front-office.

To be implemented with #708 

## Screenshots
![Image](https://user-images.githubusercontent.com/1572885/197235522-4a6838cc-77cd-4587-8151-1a62913f4d56.png)

## Implementation suggestion
```
<div class="alert alert-info">
    <i class="if if-info-circle-filled"></I>
    <div>
        <h4>Biblio search results have been optimised</h4>
        <p class="mb-2">
          To optimise Biblio's speed, we only show you the most relevant records.
          <br>Missing some results?
        </p>
        <ul class="mb-2">
          <li><strong>Refine your search query</strong> and use the <strong>filters</strong> to get a more precise set of record.</li>
          <li>Go to <a href="http://biblio.ugent.be" target="_blank">http://biblio.ugent.be</a> and <strong>use the extended search</strong> to get a complete picture.</li>
        </ul>
    </div>
</div>
```

**Comment by mietcls** [comment]
The message is currently not shown after 500 pages; did we decide to leave it out or did something go wrong?
